package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Install the latest go-ipfs binary",
   "main": "src/index.js",
   "scripts": {
-    "install": "node src/bin.js",
+    "install": "node src/bin.js $TARGET_VERSION $TARGET_OS",
```

Comment:
Is there any cross platform solution that doesn't require pulling in a dependency?